requirements.txt

```diff
@@ -1,11 +1,11 @@
-pandas
-ollama
-nltk
-bertopic
-sentence_transformers
-hdbscan
-umap
-gspread
-gspread-dataframe
-oauth2client
-ipykernel
\ No newline at end of file
+pandas==2.2.2
+ollama==0.3.2
+nltk==3.9.1
+bertopic==0.16.3
+sentence-transformers
+hdbscan~=0.8.38
+umap==0.1.1
+ipykernel===6.29.5
```

Comment:
Is the triple = intentional? from the docs: `Use of this operator is heavily discouraged and tooling MAY display a warning when it is used.`

Reply:
nope, I didn't even know it's a thing.